site/topics/transistors-gates/gates.rst

```diff
@@ -283,6 +283,115 @@ Other Gate Symbols
+
+* First, consider what that would mean
+
+    * Output ``1`` only when everything is ``1``, otherwise output ``0``
+
+
+* Following the law of associativity for and, the order that and is applied to the inputs is not important
+
+    * :math:`a \land (b \land c) = (a \land b) \land c`
+
+
+* In other words, it is possible to stack multiple and gates to create a five input and
+
+.. figure:: and_5_input_stacked.png
+    :width: 500 px
+    :align: center
+
+    Four and gates configured such that they perform an and operation on five inputs.
+
+
+* It is common to simplify this by drawing a single and gate with the desired number of inputs
+
+.. figure:: and_5_input_single_gate.png
+    :width: 333 px
+    :align: center
+
+    Single and gate with five inputs.
+
+
+* This works similarly with or gates
+* Below is an example of combining or gates to or together four inputs
+
+.. figure:: or_4_input_stacked.png
+    :width: 500 px
+    :align: center
+
+    Three or gates configured such that they perform an or operation on four inputs.
+
+
+* Like the and gate, this could be simplified as a single symbol
+
+.. figure:: or_4_input_single_gate.png
+    :width: 333 px
+    :align: center
+
+    Single or gate with four inputs.
+
+
+* Although possible to have nand/nor gates with more inputs, stacking does not work the same way
+* For example, more inputs can be added to nand gates, but stacking them the same way and gates were will not work
+
+.. figure:: nand_4_input_single_gate.png
+    :width: 333 px
+    :align: center
+
+    Single nand gate with four inputs.
+
+
+* It is important to remember these operators *mean*
+* A four input nand gate would mean a four input and with an inverted output
+
+    * :math:`\lnot(a \land b \land c \land d)`
+
+
+.. figure:: nand_4_input_stacked.png
+    :width: 500 px
+    :align: center
+
+    Stacked and gates with an inverted final output results in correct functionality for a nand gate with more than two
+    inputs.
+
+
+* Stacking three nand gates would mean
+
+    * :math:`\lnot(\lnot(a \land b) \land \lnot(c \land d))`
+
+
+.. figure:: nand_4_input_stacked_wrong.png
+    :width: 500 px
+    :align: center
+
+    Three stacked nand gates will result in functionality that is not equivalent to a proper four input nand gate. This
+    configuration will ultimately function as :math:`(a \land b) \lor (a \land b)`.
+
+
+* Following De Morgan's and the double negation laws, this simplifies to
+
+    * :math:`\lnot(\lnot(a \land b) \land \lnot(c \land d))`
+    * :math:`\lnot(\lnot((a \land b) \lor (c \land d))`
+    * :math:`(a \land b) \lor (a \land b)`
+
+
+* In other words, a four input nand gate is not functionally equivalent to stacking nand gates
+* Similar problems would exist for nor
+
+* Exclusive or (xor) gates can be stacked to have the same functionality as a single xor gate with many inputs
+* However, when xor gates have more than two inputs, they effectively become an even/odd check
```

Comment:
This first line is confusing. _a single xor gate with many inputs_ is what is trying to be defined, and "stacking" gates together does not help define it -- neither are well defined at this point. The second line actually defines what we're after, then it starts to make sense. Though it still isn't clear why "stacking" xor leads to such a result, or if that is desirable. The 3-ary xor, for example, is true when all inputs are true, which is counter-intuitive.

Reply:
I was thinking of making this part of a lab. Though, I don't really have a full plan for the labs yet. 

The trouble with the labs is, I have no sense of course pacing.